src/payment/order-details/data/constants.js

```diff
@@ -0,0 +1,15 @@
+// Product types
+export const COURSE_ENTITLEMENT_PRODUCT_TYPE = 'Course Entitlement';
```

Comment:
I'm not thrilled that these constants all live in the order-details module.  I'm thinking about how I think we would ultimately want an 'ecommerce' module that sits beside the payment-page module and contains constants and helpers for ecommerce.  Maybe the API services and sagas live there instead of in payment.  For now... they live here cause this is the only place that really needs them.